src/components/game/grid/tile/StyledTile.tsx

```diff
@@ -25,10 +25,12 @@ export const getColor = (props: { tile: ExtendedTile }): string => {
 
 const getBorderSize = (props: StyledTileProps) => (props.tile.city ? (props.isSelectedTile ? 8 : 7) : props.isSelectedTile ? 3 : 1)
 const getBorderStyle = (props: StyledTileProps) => (props.tile.city ? 'double' : 'solid')
+const getBackgroundImage = (props: StyledTileProps) => (props.tile.city ? props.tile.city.icon : '""')
 
 const hexAlpha = 99
-const applyPlayerColor = (props: StyledTileProps) =>
-    props.tile.owner
+const applyPlayerColor = (props: StyledTileProps) => {
+    const backgroundImage = getBackgroundImage(props)
+    return props.tile.owner
         ? `&:after{
```

Comment:
could be changed to ``props.tile.owner && `&:after{`` so that there is no need to return `''`